app/lib/goods_nomenclature_mapper.rb

```diff
@@ -128,4 +129,11 @@ def parent_of(goods_nomenclature)
   def heading_map?(primary, secondary)
     primary.is_a?(Heading) && secondary.is_a?(Heading)
   end
+
+  def reload_if_nil(goods_nomenclature)
+    # This is a patch to fix the error: "comparison of Integer with nil failed"
+    logger.warn("goods_nomenclature.number_indents is nil! - 'goods_nomenclature': #{goods_nomenclature}")
+
+    goods_nomenclature.reload if goods_nomenclature.number_indents.nil?
```

Comment:
since the Sentry alerts happen only a few times per day, the reload should only be triggered a few times per day too.